spring-cloud-huawei-common/pom.xml

```diff
@@ -140,5 +140,9 @@
       <artifactId>spring-cloud-starter-bootstrap</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.servicecomb</groupId>
```

Comment:
common module do not depends on service-center-cleint

Reply:
Modified